src/app/teacher/teacher-run-list-item/teacher-run-list-item.component.ts

```diff
@@ -15,13 +15,14 @@ import { ShareRunCodeDialogComponent } from '../share-run-code-dialog/share-run-
   animations: [flash]
 })
 export class TeacherRunListItemComponent implements OnInit {
+  protected animateDelay: string = '0s';
+  protected animateDuration: string = '0s';
+  protected manageStudentsLink: string = '';
+  protected periodsTooltipText: string;
   @Input() run: TeacherRun = new TeacherRun();
-
-  manageStudentsLink: string = '';
-  periodsTooltipText: string;
-  thumbStyle: SafeStyle;
-  animateDuration: string = '0s';
-  animateDelay: string = '0s';
+  @Output() runArchiveStatusChangedEvent: EventEmitter<any> = new EventEmitter<any>();
+  @Output() runSelectedStatusChangedEvent: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Looks like we don't do anything with the emitted value, so make it void?
```suggestion
  @Output() runArchiveStatusChangedEvent: EventEmitter<void> = new EventEmitter<void>();
  @Output() runSelectedStatusChangedEvent: EventEmitter<void> = new EventEmitter<void>();
```